keys/keys.go

```diff
@@ -62,7 +59,7 @@ func NodeStatusKey(nodeID int32) proto.Key {
 
 // MakeNameMetadataKey returns the key for the namespace.
 func MakeNameMetadataKey(parentID uint32, name string) proto.Key {
-	k := make([]byte, 0, len(NameMetadataPrefix)+maxVarintSize+len(name))
+	k := make([]byte, 0, len(NameMetadataPrefix)+binary.MaxVarintLen64+len(name))
```

Comment:
Will look at taking that out in a new PR

On Tue, Jun 16, 2015 at 10:14 AM Peter Mattis <notifications@github.com>
wrote:

> In keys/keys.go
> <https://github.com/cockroachdb/cockroach/pull/1356#discussion_r32524472>:
>
> > @@ -62,7 +59,7 @@ func NodeStatusKey(nodeID int32) proto.Key {
> >
> >  // MakeNameMetadataKey returns the key for the namespace.
> >  func MakeNameMetadataKey(parentID uint32, name string) proto.Key {
> > -	k := make([]byte, 0, len(NameMetadataPrefix)+maxVarintSize+len(name))
> > +	k := make([]byte, 0, len(NameMetadataPrefix)+binary.MaxVarintLen64+len(name))
>
> The use of binary.PutVarint in util/encoding.Encode should really be
> removed. Looks like the only use of that method is by engine.Increment()
> which appears to be unused.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/1356/files#r32524472>.
>